src/dragonball/src/vm/aarch64.rs

```diff
@@ -138,6 +103,50 @@ impl Vm {
         Ok(())
     }
 
+    /// Generate fdt information about VM.
+    fn get_fdt_vm_info<'a>(
+        &'a self,
+        vm_memory: &'a GuestMemoryImpl,
+        cmdline: &'a str,
+        initrd_config: Option<&'a InitrdConfig>,
+        vcpu_manager: &'a MutexGuard<VcpuManager>,
+    ) -> FdtVmInfo {
+        let guest_memory = vm_memory.memory();
+        let vcpu_mpidr = vcpu_manager
+            .vcpus()
+            .into_iter()
+            .map(|cpu| cpu.get_mpidr())
+            .collect();
+        let vcpu_boot_onlined = vec![];
+        let vpmu_feature = vcpu_manager.vpmu_feature();
+        // This configuration is used for passing cache information into guest.
+        // TODO: dragonball-sandbox #274; kata-containers #6969
+        let cache_passthrough_enabled = false;
+        let fdt_vcpu_info = FdtVcpuInfo::new(
+            vcpu_mpidr,
+            vcpu_boot_onlined,
+            vpmu_feature,
+            cache_passthrough_enabled,
```

Comment:
Out of curiosity, what's cache_passthrough?

Reply:
This is used as a switch to pass cache information into guest. See #6969.

Reply:
OK